sycl/unittests/kernel-and-program/KernelBuildOptions.cpp

```diff
@@ -109,13 +109,39 @@ TEST(KernelBuildOptions, KernelBundleBasic) {
   sycl::kernel_bundle KernelBundle =
       sycl::get_kernel_bundle<sycl::bundle_state::input>(Ctx, {Dev},
                                                          {KernelID});
+  try {
+    // no supported properties now
```

Comment:
```suggestion
    // unsupported property.
```

Reply:
fixed in https://github.com/intel/llvm/pull/15647/commits/808287ac7f44524821db9d33b84c4c807534422b